.github/workflows/npm-publish.yml

```
@@ -1,15 +1,23 @@
   push:
     tags:
       - 'v*.*.*'
 
+permissions:
+  contents: write
+
 jobs:
   publish-npm:
     runs-on: ubuntu-latest
+    env:
+      RELEASE_ARTIFACTS: ${{ runner.temp }}/editor-for-vue-release-artifacts
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)